### PR TITLE
mupdf 1.12.0

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/mupdf.rb [hunk 1]
```diff
@@ -36,14 +22,6 @@ def install
     # Work around bug: https://bugs.ghostscript.com/show_bug.cgi?id=697842
     inreplace "Makerules", "RANLIB_CMD := xcrun", "RANLIB_CMD = xcrun"
```
Comment (fxcoudert) [inline Formula/mupdf.rb]: If we have this work-around, we could thus remove `HAVE_GLFW=no` below

Comment (ilovezfs) [reply Formula/mupdf.rb]: won't we have to put that back later?

Comment (fxcoudert) [reply Formula/mupdf.rb]: Don't think so.

Comment (fxcoudert) [reply Formula/mupdf.rb]: Upstream fixed the bug, actually, so we need to remove the `inreplace` too

**Comment by DomT4** [comment]
Thanks for picking these up @ilovezfs ❤️

**Pull request closed without merging**